astropy/io/fits/card.py

```diff
@@ -235,7 +235,7 @@ def keyword(self, keyword):
             keyword = keyword.rstrip()
             keyword_upper = keyword.upper()
             if (len(keyword) <= KEYWORD_LENGTH and
-                self._keywd_FSC_RE.match(keyword_upper)):
+                    self._keywd_FSC_RE.match(keyword_upper)):
```

Comment:
slight preference, but I prefer to let way for W503 than W504, so break the line before the operator

Reply:
The way to break lines with respect to the operator could be changed, but there are a lot of those, too many for me tonight. 
And even if you break before the operator the newline must be indented.

Reply:
yes, the indent is needed. It's not a strong preference from my side, so perfectly all right if you  prefer not to change  it, I'm happy to merge as is (after it made consistent thorough as at some places it's already after the linebreak)